modules/pinger/src/main/java/org/hawkular/component/pinger/PingManager.java

```diff
@@ -53,7 +59,8 @@
     // How long do we wait between each round
     private static final int WAIT_MILLIS = 500;
 
-    String tenantId = "test";
+    private final String tenantId = "test";
```

Comment:
someone has the create the default tenant and environment. This should be probably done by the keycloak ui code, this is just to make it working w/ the 'end-to-end' test